tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,149 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations, unlimited metrics retention and global querying across clusters.
+
+Thanos builds upon existing Prometheus instances which makes it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will be using one Prometheus Server.
+
+Let's start this initial Prometheus setup, ready?
+
+
+## Generate Artifical Metric Data
+
+Execute the following command:
+
+```
+mkdir -p test
+```{{execute}}
+
+```
+docker run -i dockerenginesonia/thanosbench:v7 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -v /root/test:/test -i  dockerenginesonia/thanosbench:v7 block gen --output.dir test
+```{{execute}}
+
+## Prometheus Configuration Files
+
+Here, we will prepare configuration files for the Prometheus instance.
```

Comment:
```suggestion
Here, we will prepare configuration files for the Prometheus instance that will run with our pre-generated data.
It will also scrape our components we will use in this tutorial.
```